pom.xml

```diff
@@ -49,7 +49,7 @@
     </parent>
 
     <properties>
-        <uk.bom.version>1.1.4-SNAPSHOT</uk.bom.version>
+        <uk.bom.version>1.1.6-SNAPSHOT</uk.bom.version>
```

Comment:
Update the bom version to 1.1.7-SNAPSHOT, I have merge and updated the minor version this morning to introduce the intentType enum for VRP payments.

Reply:
done